app/mailers/application_mailer.rb

```diff
@@ -1,6 +1,8 @@
 # here you can use and Env var in from's variable
 class ApplicationMailer < ActionMailer::Base
   SUPPORT_ADDRESS = 'support@rootstrap.com'.freeze
+  SUPPORT_DOMAIN =  SUPPORT_ADDRESS.gsub(/.+@([^.]+).+/, '\1')
```

Comment:
would be useful to have also the TLD, too? ("rootstrap" + ".com")

Reply:
done